lib/index.js

```
@@ -12,7 +12,9 @@ const Wreck = require('wreck');
 const internals = {
     defaults: {
         threshold: 20,
+        errorThreshold: 0,
         schema: 'good-http',
+        ignoreRequestErrors: false,
```

I kept `ignoreRequestErrors` because `errorThreshold` only supports ignoring a finite amount of errors, but not all errors. Is "ignore all errors" a use case you'd like to support? If so, we can still remove `ignoreRequestErrors` by making `errorThreshold: -1` mean "ignore all errors".